Interstation-Two-WW2/code/modules/mob/inventory.dm

```diff
@@ -143,6 +143,14 @@ var/list/slot_equipment_priority = list( \
 	W.layer = initial(W.layer)
 	W.dropped()
 	return 0
+
+//Checks if mob has one or more empty hands, depending on input param:
+//	both: if 0 (false), it returns true if at least one hand is free, otherwise 1 for checking both
+/mob/proc/has_empty_hand(var/both = 0)
```

Comment:
Yeah. If I'm not crazy and there is a proc that handles this already it can be replaced later.